Selection.js

```diff
@@ -285,6 +285,12 @@ define([
 			//		except that clicks/keystrokes without modifier keys will clear
 			//		the previous selection.
 
+			// If the target is already selected, and is the sole selection, ignore a user action
+			// that would simply select the target (causing unnecessary deselect/select).
+			if (!event[ctrlEquiv] && this.isSelected(target) && this.getSelectedCount() === 1) {
```

Comment:
On the line below where we're checking control the logic used is
`!(event.keyCode ? event.ctrlKey : event[ctrlEquiv])`

Should we be replicating the same check here?

Reply:
I can't think of why that logic would be used anywhere. It only makes sense to me if:

1. Mac browsers typically don't supply a value for `event.keyCode`
2. There is a specific browser for which `has('mac')` is `true`, and that browser _does_ supply a value for `event.keyCode`, and instead of `event.metaKey` it sets `event.ctrlKey` when the Command key is pressed

I haven't found any discussion of ctrlKey/metaKey that justifies this logic. Am I missing something?

Reply:
You're right, I don't know of any situation where that makes sense.